### ISSUE TITLE
Getting error in create build

### ISSUE THREAD

**Issue by anandharaj-dotworld** [Genymobile/scrcpy#2793]
```
./run x
INFO: scrcpy 1.19 <https://github.com/Genymobile/scrcpy>
x/server/scrcpy-server: 1 file pushed. 1.8 MB/s (37382 bytes in 0.019s)
WARN: Killing the server...
CRITICAL: Requested to kill 0, this is an error. Please report the bug.
./run: line 23: 352248 Aborted                 SCRCPY_SERVER_PATH="$BUILDDIR/server/scrcpy-server" "$BUILDDIR/app/scrcpy" "$@"
```
what was i mising

**Comment by rom1v** [comment]
Did you do any modification to the source code before building?

**Comment by rom1v** [comment]
Does it always happen? Could yoj reproduce in 1.20?

**Comment by anandharaj-dotworld** [comment]
Yes, i running a server using the following command 
```
CLASSPATH=/data/local/tmp/classes.dex app_process / com.genymobile.scrcpy.Server "1.19" "verbose" 8  "999999999"  "20"   "1" "true" "9999:9999:0:0"    "true"  "true"  "0"  "true"   "-" "-"  "-" "false"
```
I need to edit a source. can you help me to edit the code.

**Comment by anandharaj-dotworld** [comment]
Let me know your skype or any other contact details. We will discuss about this.

**Comment by rom1v** [comment]
> ```
> WARN: Killing the server...
> CRITICAL: Requested to kill 0, this is an error. Please report the bug.
> ```

Indeed, in v1.19, when the server failed to start, the client code still attempted to kill it, which was an error. It has been solved by the refactors done in v1.20.

Now, in v1.20, the client will not assert/crash.

However, `app_process` may still crash on the device (the `Aborted` you see in the logs).

Try to start the command manually (independent of scrcpy):
```bash
adb shell 'CLASSPATH=… app_process …'
```

Typically, it aborts that way if it does not find the class you want to run.

> `CLASSPATH=/data/local/tmp/classes.dex`

Maybe try to rename to `.jar`: 83ace842809034f8076296a338c017e49519f068 (but it should work also with `.dex`).
Verify that the file exists and has correct permissions on the device.

Try to execute manually with the exact same values as scrcpy pass…

**Comment by anandharaj-dotworld** [comment]
I trying to connecting to build a pre build server. exactly i need i make one android reverse reverse shell application and it was listening in netcat. now i trying to connect that device actually. Let me know your skype id for further discussion

**Comment by rom1v** [comment]
> Let me know your skype id for further discussion

Sorry, I have no time to give support for people's own projects.

I often answer to questions on issues (like I did just above), but I will not investigate more if this is not a problem in scrcpy itself.
